File: Rich_wealth.java
import java.util.stream.IntStream;

public class Rich_wealth {
    public static void main(String[] args) {
        Solution sol = new Solution();

        int[][] accounts = {{1,2,3},{3,2,1}};

        System.out.println(sol.maximumWealth(accounts));
    }
    
}

class Solution {
    public int maximumWealth(int[][] accounts) {
        int max_wealth = 0;

        for (var i : accounts) {
            max_wealth = Math.max(max_wealth, IntStream.of(i).sum());
        }
        return max_wealth;
    }
}

File: Rich_wealth.rb
### Richest Customer Wealth

# @param {Integer[][]} accounts
# @return {Integer}
def maximum_wealth(accounts)
    max_wealth = 0
    
    accounts.each do |sub|
        max_wealth =  [sub.sum, max_wealth].max
    end
    
    return max_wealth
end

accounts = [[1,2,3],[3,2,1]]
puts maximum_wealth(accounts)